erpnext/stock/report/stock_balance/stock_balance.py

```diff
@@ -72,17 +79,17 @@ def get_columns():
 		_("Description")+"::140",
 		_("Warehouse")+":Link/Warehouse:100",
 		_("Stock UOM")+":Link/UOM:90",
-		_("Opening Qty")+":Float:100",
+		{"label": _("Opening Qty"), "fieldname": "opening_qty", "fieldtype": "Float", "width": 100, "convertible": "qty"},
```

Comment:
+1 for using the new format, but why not refactor all the columns, so that it looks consistent